@PestoOptions/PestoOptions.m

```diff
@@ -419,9 +419,9 @@
             end
             
             % Add required subclasses
-            obj.MCMC = PestoSamplingOptions();
-            obj.HO = HOOptions();
-            obj.HO.foldername = obj.foldername;
+%             obj.MCMC = PestoSamplingOptions();
+%             obj.HO = HOOptions;
+%             obj.HO.foldername = obj.foldername;
```

Comment:
Hi,
I'm not sure, but you probably want to leave those lines in here...
Seems like this is taken from an old fix, sorry.